<launch>
<machine name="slave" address="172.20.10.10" user="pi" password="321" env-loader="/home/pi/kotyambaCar/src/catkin-ws/install/env.sh" />
<node pkg="kotyambaCar" type="command_listener.py" name="command_listener" machine="slave"/>
<node pkg="kotyambaCar" type="ultrasonic_sensor.py" name="ultrasonic_sensor" machine="slave" >
  <param name="frequency" value="10.0" />
</node>
</launch>
